src/Components/DemographicSurvey.js

```diff
@@ -9,9 +11,18 @@ const myCss = {
 
 function DemographicSurvey({ surveyModel }) {
     const [results, setResults] = useState('');
+    const [cacheScreen, setCacheScreen] = useLocalStorage('screenerSurvey', null);
```

Comment:
This hook is creating a new empty local storage record, so it doesn't have access to the uuid we need. The survey data record already exists as `cache` in `ScreenerSurvey`, so we can pass that down as a prop from `ScreenerSurvey` -> `OutcomeComponent` -> `DemographicSurvey`. Then we can use `cache.uuid` on lines 18 and 20.

For `cacheDem` below this, I personally don't think we need it since you can never return to an incomplete dem survey without going through the whole survey again, but maybe there's something else I'm not thinking of

Reply:
An alternative to passing down `cache` - we can just grab it from local storage the plain way `localStorage.getItem('surveyData')`

Reply:
An alternative to passing down `cache` - we can just grab it from local storage the plain way `localStorage.getItem('screenerSurvey')`. But we'll just need to remember to update the key name in both places if it ever changes.

Reply:
Hey sorry ignore the above! I looked at the source code and ran it again and this works.